spec/actors/hyrax/actors/file_set_actor_spec.rb

```diff
@@ -3,345 +3,350 @@
 RSpec.describe Hyrax::Actors::FileSetActor do
   include ActionDispatch::TestProcess
 
-  let(:user)          { create(:user) }
-  let(:file_path)     { File.join(fixture_path, 'world.png') }
-  let(:file)          { fixture_file_upload(file_path, 'image/png') } # we will override for the different types of File objects
-  let(:local_file)    { File.open(file_path) }
-  let(:file_set)      { create(:file_set, content: local_file) }
-  let(:actor)         { described_class.new(file_set, user) }
-  let(:relation)      { :original_file }
-  let(:file_actor)    { Hyrax::Actors::FileActor.new(file_set, relation, user) }
-
-  describe 'private' do
-    let(:file_set) { build(:file_set) } # avoid 130+ LDP requests
-
-    describe '#assign_visibility?' do
-      let(:viz) { Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC }
-
-      it 'without params, returns false' do
-        expect(actor.send(:assign_visibility?)).to eq false
-      end
-      it 'with string-keyed or symbol-keyed visibility, returns true' do
-        expect(actor.send(:assign_visibility?, visibility: viz)).to eq true
-        expect(actor.send(:assign_visibility?, 'visibility' => viz)).to eq true
+  [false, true].each do |use_valkyrie|
```

Comment:
This spec hasn't really changed.  Adding the line to run all tests with use_valkyrie=true and again with false, changed the indentation, which makes it look like the file had substantial changes when it didn't.